tests/testdemographics.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * @fileoverview Defines tests for audience demographics.
+ */
+
+
+/** @const {!RegExp} */ var PATTERN = /^\d([,\.]*\d)*%$/;
+
+/** @const {!Object.<string, function(NodeList):boolean>} */ var VALIDATORS = {
+  'gender-data-table': function(elements) {
+    return testStaticDataTable(elements, ['male', 'female']);
+  },
+  'browsing-location-data-table': function(elements) {
+    return testStaticDataTable(elements, ['home', 'school', 'work']);
+  },
+  'geo-data-table': function(elements) {
+    /** @type {number} */ var length = elements.length;
+    /** @type {number} */ var index = 0;
+    /** @type {string} */ var content;
+    /** @type {NodeList} */ var cells;
+    /** @type {Node} */ var parentNode =
+        document.getElementById('geo-data-table') &&
+        document.getElementById('geo-data-table').parentNode;
+
+    if (parentNode.className.indexOf('true') >= 0 && elements.length){
```

Comment:
- If `parentNode` is `null`?
- Suggestion: `~parentNode.className.indexOf('true')`;
- Missed space between `)` and `{`.
- Please format code according to style guide.